docs/config.json

```diff
@@ -419,13 +419,33 @@
       "title": "Manage your Cluster",
       "entries": [
         {
-          "title": "Admin Guides",
-          "slug": "/management/admin/",
+          "title": "Registering Teleport Services",
```

Comment:
Personally, if I were looking for this doc, I'd be searching for something with "join" in the title. Do you think "registering services" is descriptive enough?